src/js/dom.js

```diff
@@ -0,0 +1,59 @@
+
+export function appendToMain($element) {
+    const $main = document.getElementById('app');
+    $main.appendChild($element);
+    
+}
+
+function hidePokemonUI() {
```

Comment:
I think it would be more proper if we name this function `hidePokemonInformation` or `hidePokemonStats`. What do you think?

Reply:
I had my doubts about this name since it covers various fields of a single pokemon, including its stats. hidePokemonInformation should work best

Reply:
I had my doubts about this name since it covers various fields of a single pokemon, including its stats. hidePokemonInformation should work better